identify/interpreters.py

```diff
@@ -6,6 +6,7 @@
     'bash': {'shell', 'bash'},
     'csh': {'shell', 'csh'},
     'dash': {'shell', 'dash'},
+    'nix-shell': {'shell', 'bash'},
```

Comment:
iirc (it was proposed before) this doesn't actually work since `nix-shell` could be any interpreter

Reply:
I think you are referring to the cases of https://github.com/chriskuehl/identify/issues/128 where `-i <interpreter>`? Afaik, `nix-shell` does always invoke a bash environment. Wold this addition conflict with #128? @roberth, you are more of an expert on the subject, could you clarify?

Reply:
nah this was a long time ago, nix does a strange thing where the actual language is on the second line of the file and the file is not always executable.  I can't find the issue, it appears it wasn't on this project but perhaps on one of the nix projects

Reply:
https://gist.github.com/travisbhartwell/f972aab227306edfcfea